extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/KubernetesTaskRunner.java

```diff
@@ -263,11 +263,14 @@ public void shutdown(String taskid, String reason)
       return;
     }
 
+    workItem.shutdown();
+    if (!workItem.getResult().isDone()) {
+      return;
```

Comment:
Should there be an info log here saying something like "Attempting to shutdown a task %taskId% that is not yet shutdown."

Reply:
i think its okay, there are already a lot of logs when this happens. during my testing i was able to follow what had happened pretty easily